bootstrap.sh

```
@@ -1,10 +1,24 @@
 #!/bin/bash
+
+echo "Creating cluster..."
+kind create cluster --config cluster.yml
```

Good: the script uses `kind create cluster --config cluster.yml` which matches the requirement to create the cluster from cluster.yml. Consider adding `set -euo pipefail` near the top (after the shebang) and a quick check that `cluster.yml` exists to fail early and make the script more robust.